### DISCUSSION TITLE
请问支持vue3吗

### DISCUSSION THREAD

**Discussion by qq1037305420** [Tencent/Hippy#4008] [General]
请问支持vue3吗？

**Comment by qq1037305420** [comment]
请问有支持纯血鸿蒙的规划吗

**Comment by qq1037305420** [comment]
> 请问支持vue3吗？

文档上没写但是看到有vue-next包

**Reply by gguoyu** [reply]
有的，
Hippy 2.x 的 vue3 文档 https://hippyjs.org/#/hippy-vue/vue3
Hippy 3.x 的 vue3 文档 https://doc.openhippy.com/#/api/hippy-vue/vue3
